vimrc

```diff
@@ -272,6 +272,15 @@ endfunction
 command! -nargs=0 GitGrepWord :call GitGrepWord()
 nnoremap <silent> <Leader>gw :GitGrepWord<CR>
 
+function! GitWebURL() range
```

Comment:
This should probably be `GitHubWebURL` given that it is GitHub specific.

Reply:
Good point. I renamed it to `GitHubURL`: d500706